### PR TITLE
Update starting-a-project.md

### PR DESCRIPTION
**Pull request by mohammadsaeed-dev**
- [ ] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _articles/starting-a-project.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
```
Comment (Aof1983) [inline _articles/starting-a-project.md:1]: #5508

**Review by Aof1983 — CHANGES_REQUESTED** [review]
Duplicate of #

**Comment by YoYBaBy** [comment]
讓我考慮一下 謝謝





發自 iPhone 版 Yahoo 郵箱
星期二, 四月 20, 2021, 22:48 於 Aof1983 ***@***.***>寫道：
@Aof1983 requested changes on this pull request.
Duplicate of #

**Comment by ArtNow198** [comment]
> * [ ]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [ ]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Pull request closed without merging**